conanfile_installer.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import os
+from conanfile_base import ConanfileBase
+
+
+class ConanfileInstaller(ConanfileBase):
+    name = "flex_installer"
+    version = ConanfileBase.version
+    exports = ConanfileBase.exports + ["conanfile_base.py"]
+
+    settings = "os_build", "arch_build", "compiler"
```

Comment:
I think you need to add `arch` and remove it on package id. When cross-compiling, Conan won't change the compiler command -m<arch> without arch on settings. Actually this is a Conan limitation.

Reply:
added